source/base/Text.ooc

```diff
@@ -174,6 +174,14 @@ Text: cover {
 		this free(Owner Receiver)
 		result
 	}
+	print: func {
+		printf("%s", this _buffer _backend pointer as Char*)
```

Comment:
One can do that without callbacks by reopening `stdout` with `freopen`, you can redirect anything that goes to stdout anywhere you like. Are you sure we should bind the Debug module into `Text` ? `String` class didn't use Debug in `print`.